docs/ref/migration-operations.txt

```diff
@@ -202,21 +202,7 @@ is set, its column name).
 Creates an index in the database table for the model with ``model_name``.
 ``index`` is an instance of the :class:`~django.db.models.Index` class.
 
-For example, to add an index on the ``title`` and ``author`` fields of the
-``Book`` model::
-
-    from django.db import migrations, models
-
-    class Migration(migrations.Migration):
-        operations = [
-            migrations.AddIndex(
-                'Book',
-                models.Index(fields=['title', 'author'], name='my_index_name'),
-            ),
-        ]
-
-If you're writing your own migration to add an index, you must assign a
-``name`` to the ``index`` as done above.
+It only accepts an `Index` instance for which the `name` attribute is defined.
```

Comment:
Not sure this is important to mention since most users shouldn't encounter it and a helpful message is raised in that case. Otherwise, you need to use double backticks around "Index" and "name" to have an effect on the rendered docss.